### ISSUE TITLE
Traceback, OS Error [WinError 193] 1% is not a valid Win32 application

### ISSUE THREAD

**Issue by MrJoJI110** [cgvirus/Blender-Instant-Mesh-Bridge-addon#5]
I am gettnig this error in blender 3.3.0.
[Traceback](https://imgur.com/a/NdIUoma)
Also Note: Instant Remesher works fine if i open it manually without the addon, but whenever I use the addon to do it, it gives this error

**Comment by MrJoJI110** [comment]
Figured it out,
I am using blender 3.3.0 downloaded from Windows Store, Because of it, Blender is installed inside C/Program File/WindowsApp/BlenderFoundation. The problem is that WindowsApp folder is protected by System User and is probably restricting subprocess from launching Instant meshes. I have tested it with a seperate installation of Blender and it works fine. Will it again by moving Instant meshes.exe to ./WindowsApp
